### PR TITLE
Metabolism and Spirit Bottle updates

### PR DESCRIPTION
**Pull request by jd0yle**
Added descriptions and rules for No Metabolism, Undead Metabolism, and Spirit Bottles

### CODE REVIEW

File: rules-topics/16_life-and-death.md [hunk 1]
```diff
@@ -88,3 +88,28 @@ Killing Blow is a 3-count that can be interrupted. When Killing Blowed, the targ
...
 
 Dismembering a character for the purpose of attempting to use a smaller body part to heal, revive, or even hide the character is not any different than if the character were not dismembered. If a character is **Bleeding Out** or **Dead** and someone wishes to bring that character to a healer, or move the body away from where it fell, they may role-play carrying the body by picking it up on a three count _(ex: "One I pick you up… Two I pick you up… Three I pick you up…")_ and carrying it elsewhere. The carrying player may not run, and both players must roleplay maintaining contact and remain within arm's reach of each other. If any effect would forcibly move the carrier away at a pace faster than a walk, or would otherwise render them unable to hold game items in their hands, the character must drop the body on a One Count _(ex: "One I drop you")_ and may only resume carrying them on a new three count.
+
+## Metabolisms
```
Comment (sd0yle) [inline rules-topics/16_life-and-death.md:92]: Can we either update or remove the Metabolism section in the Status Effects document?  Several clarifications in the past were put there and we had a placeholder for Metabolism.

File: rules-topics/16_life-and-death.md [hunk 2]
```diff
@@ -88,3 +88,28 @@ Killing Blow is a 3-count that can be interrupted. When Killing Blowed, the targ
...
+Most people and creatures have a regular metabolism. Characters with "no metabolism" or "undead metabolism" are affected by some game effects differently.
+
+### No Metabolism
+- Crumble at 0 Body
+- Immune to Toxin
+- Immune to Berserk, Command: Confuse, Death, Boon of Nature, Drain, First Aid, Heal Wounds, Hearthstone, Infect, Life, Regenerate, Revive, Revivify, Sleep, Stun, Vampiric Touch, Waylay
```
Comment (sd0yle) [inline rules-topics/16_life-and-death.md:98]: Please add a clarification about if a character with no metabolism is Immune to receiving these effects/items/skills or if they are immune period and cannot benefit from them even if used by themselves.  ie:  if a no metabolism character casts Vampiric Touch, can they be healed?

File: rules-topics/16_life-and-death.md [hunk 3]
```diff
@@ -88,3 +88,28 @@ Killing Blow is a 3-count that can be interrupted. When Killing Blowed, the targ
...
+
+> _Generally, a creature that has no metabolism cannot be healed by typical means, but will have an alternate method of being healed. For instance, a Fire Elemental would be immune to standard healing effects, but could be healed by Flame._ 
+
+
+### Undead Metabolism
+Same effects as per "No Metabolism", except the character is affected normally by Heal Wounds. They are also affected by effects and abilities that specifically target Undead (such as Harm Undead).
```
Comment (sd0yle) [inline rules-topics/16_life-and-death.md]: Please clarify "same effects" here?  Does this include the "Immune to Berserk ... [ ] Waylay"?  If yes, same clarification request about "receiving" versus self-use.

File: rules-topics/16_life-and-death.md [hunk 4]
```diff
@@ -88,3 +88,28 @@ Killing Blow is a 3-count that can be interrupted. When Killing Blowed, the targ
...
+
+## Spirit Bottle
+A character with a Spirit Bottle has stored their spirit for safe keeping in an item. Doing so protects the character from incurring Strain, however, it causes the character's body to become brittle. All characters with a Spirit Bottle must crumble at 0 body, and unless otherwise specified, have No Metabolism.
+
+Instead of resurrecting normally, a character with a Spirit Bottle may choose to reform at their Spirit Bottle. Doing so follows the rules for normal  resurrection, with these exceptions:
```
Comment (sd0yle) [inline rules-topics/16_life-and-death.md]: Extra space between "normal" and "resurrection"

**Review by sd0yle — CHANGES_REQUESTED** [review]

File: rules-topics/16_life-and-death.md (hunk 3 above)
Comment (jd0yle) [reply rules-topics/16_life-and-death.md]: I don't understand, it's the same list of effects as "No Metabolism"

It seemed silly to explicitly duplicate the list of effects.

File: rules-topics/16_life-and-death.md (hunk 1 above)
Comment (jd0yle) [reply rules-topics/16_life-and-death.md:92]: I couldn't find a reference to metabolism in the status effects chapter.

File: rules-topics/16_life-and-death.md (hunk 2 above)
Comment (jd0yle) [reply rules-topics/16_life-and-death.md:98]: No, they're immune to it?

**Comment by jd0yle** [comment]
@sd0yle Made some edits, please review?

File: rules-topics/16_life-and-death.md (hunk 1 above)
Comment (sd0yle) [reply rules-topics/16_life-and-death.md:92]: Used to be a placeholder near the top, it may be removed now.

**Review by sd0yle — APPROVED** [review]

**Pull request merged**